my-app/src/App.js

```diff
@@ -20,6 +22,12 @@ function App() {
       <div className="content">
         <ContentList content={content} />
       </div>
+      <Section category={'business'} title={'BUSINESS'}/>
+      <Section category={'technology'} title={'TECHNOLOGY'}/>
+      <Section category={'health'} title={'HEALTH'}/>
+      <Section category={'sports'} title={'SPORTS'}/>
+      <Section category={'science'} title={'SCIENCE'}/>
+      <Section category={'entertainment'} title={'ENTERTAINMENT'}/>
```

Comment:
or
```jsx
 { category.map((cat) => <Section category={cat} title={cat.toUpperCase()}/>)}
 ```
 your `category` can be defined as an array or coming from an option/select list